services/src/main/java/org/keycloak/protocol/oid4vc/issuance/OID4VCIssuerWellKnownProvider.java

```diff
@@ -115,4 +130,30 @@ public static String getIssuer(KeycloakContext context) {
     public static String getCredentialsEndpoint(KeycloakContext context) {
         return getIssuer(context) + "/protocol/" + OID4VCLoginProtocolFactory.PROTOCOL_ID + "/" + OID4VCIssuerEndpoint.CREDENTIAL_PATH;
     }
+
+    private static final String VC_KEY = "vc";
```

Comment:
AFAIK, there are three points where "vc" is defined:
- OID4VCIssuerWellKnownProvider (here)
- OID4VCClientRegistrationProvider
- JwtSigningService

It might be better to have only one "vc".